### DISCUSSION TITLE
No HRR with liquid fuel

### DISCUSSION THREAD

**Discussion by MNJOLE** [firemodels/fds#12778] [Q&A]
Hello, I am currently studying the implementation of heptane liquid fuel as a pyrolysis model.
However, despite confirming that the temperature of the heptane pool is increasing, ignition is not occurring. 
What input value am I missing? 
I appreciate any advice or hints.

------------------------
&MESH ID='Mesh01', IJK=27,27,67, XB=-0.060606,1.575758,-0.060606,1.575758,-0.1,4.0/

&REAC ID='FDS6 N-HEPTANE',
      FYI='FDS6 Predefined',
      FUEL='N-HEPTANE',
      CO_YIELD=0.01,
      SOOT_YIELD=0.037,
      HEAT_OF_COMBUSTION=4.13E+4,
      RADIATIVE_FRACTION=0.33/

&DEVC ID='GAS', QUANTITY='TEMPERATURE', XYZ=0.775679,0.729742,0.025/

&MATL ID='heptane',
      SPECIFIC_HEAT=2.24,
      CONDUCTIVITY=0.14,
      DENSITY=675.0,
      ABSORPTION_COEFFICIENT=333.0,
      EMISSIVITY=1.0,
      HEAT_OF_COMBUSTION=439.0,
      HEAT_OF_REACTION=317.0,
      BOILING_TEMPERATURE=98.35/

&SURF ID='heptane liquid',
      COLOR='YELLOW',
      BURN_AWAY=.TRUE.,
      MATL_ID(1,1)='heptane',
      MATL_MASS_FRACTION(1,1)=1.0,
      THICKNESS(1)=0.025/

&SURF ID='hot block',
      RGB=255,102,0,
      TMP_FRONT=1000.0,
      RAMP_T='hot block_RAMP_T',
      EMISSIVITY=1.0/
&RAMP ID='hot block_RAMP_T', T=0.0, F=0.0/
&RAMP ID='hot block_RAMP_T', T=1.0, F=1.0/
&RAMP ID='hot block_RAMP_T', T=30.0, F=1.0/
&RAMP ID='hot block_RAMP_T', T=31.0, F=0.0/

**Comment by rmcdermo** [comment]
Looks like you have not included the SPEC_ID='N-HEPTANE' in your MATL decomposition.  Have a look at this example:

https://github.com/firemodels/fds/blob/master/Validation/Pool_Fires/FDS_Input_Files/heptane_1_m.fds

**Reply by MNJOLE** [reply]
I really appreciate it. It seems like I always overlook these important input values when working with Pyrosim. Thank you for the prompt response.
